### ISSUE TITLE
Saving Feed Items

### ISSUE THREAD

**Issue by chf2** [marcusegues/newsreader#4]
I would ask that we think through exactly when we want to save feed items. If we save them all automatically we may end up with a ton of data in the database. Pruning old records is definitely doable but might require something like a rake task which may be moderately more advanced to put together. More importantly, it is not clear to me when we will fetch feed items from the database as opposed to dynamically displaying them from our front-end application.

In other words, will we ever pull feed items out of the DB directly for a given feed as opposed to grabbing the latest feed items from the url?
